.circleci/config.yml

```diff
@@ -1,41 +1,132 @@
-version: 2
+version: 2.1
+defaults: &defaults
+  working_directory: ~/user-data-for-fraud-prevention
+  docker:
+    - image: circleci/node:10.15.3-browsers
+
 orbs:
```

Comment:
Could the location of orbs be a problem? Before your changes, It was placed at the beginning of the file. Not sure if it is an issue but just wondering..